src/Symfony/Component/Yaml/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 2.8.0
 -----
 
+ * Deprecated usage of @ and ` at the beginning of an unquoted string
```

Comment:
Only as long as you don't add another one in the file. They handle it only because they cannot find the corresponding one.